docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -2,3 +2,349 @@
+     244  1s ago     PwCalculation             ⏵ Waiting        Monitoring scheduler: job state QUEUED
+
+    Total results: 5
+
+    Info: last time an entry changed state: 0s ago (at 12:45:40 on 2020-11-25)
+
+While you wait for the workflow to complete, let's start exploring its provenance.
+
+The full provenance graph obtained from ``verdi node graph generate`` will already be rather complex (you can try!), so let's try browsing the provenance interactively instead.
+
+Start the AiiDA REST API:
+
+.. code-block:: console
+
+    $ verdi restapi
+
+and open the |provenance browser| (from the browser inside the virtual machine).
+
+.. |provenance browser| raw:: html
+
+    <a href="https://www.materialscloud.org/explore/ownrestapi?base_url=http://127.0.0.1:5000/api/v4" target="_blank">Materials Cloud provenance browser</a>
+
+.. note::
+
+    The provenance browser is a Javascript application that connects to the AiiDA REST API.
+    Your data never leaves your computer.
+
+.. some general comment on importance of the graph?
+.. a sentence on how to continue from here
+
+Browse your AiiDA database:
+
+    * Start by finding your `Quantum ESPRESSO`_ calculation (the type of node is called a ``CalcJobNode`` in AiiDA, since it is run as a job on a scheduler).
+    * Select ``Calculations`` in the left menu to filter for calculations only.
+    * Inspect the raw inputs and outputs of the calculation, and use the provenance browser to explore the input and output nodes of the calculation and the whole provenance of your simulations.
+
+.. note::
+
+    When perfoming calculations for a publication, you can export your provenance graph using ``verdi export create`` and upload it to the `Materials Cloud Archive`_, enabling your peers to explore the provenance of your calculations online.
+
+Once the workchain is finished, use ``verdi process show <PK>`` to inspect the ``PwBandStructureWorkChain`` and find the PK of its ``band_structure`` output.
+Use this to produce a PDF of the band structure:
+
+.. code-block:: console
+
+   $ verdi data bands export --format mpl_pdf --output band_structure.pdf <PK>
+
+.. figure:: include/images/si_bands.png
+   :width: 100%
+
+   Band structure computed by the ``PwBandStructure`` workchain.
+
+.. note::
+   The ``BandsData`` node does contain information about the Fermi energy, so the energy zero in your plot will be arbitrary.
+   You can produce a plot with the Fermi energy set to zero (as above) using the following code in e.g. a Jupyter notebook:
+
+   .. code-block:: ipython
+
+        %matplotlib inline
+        import aiida
+        aiida.load_profile()
+
+        from aiida.orm import load_node
+
+        scf_params = load_node(<PK>)  # REPLACE with PK of "scf_parameters" output
+        fermi_energy = scf_params.dict.fermi_energy
+
+        bands = load_node(<PK>)  # REPLACE with PK of "band_structure" output
+        bands.show_mpl(y_origin=fermi_energy, plot_zero_axis=True)
```

Comment:
Should this also work if I just copy paste it (and replace pks) into the verdi shell remotely? I tried this and it didn't work for me.

Reply:
Hmm, let me try it. Won't be an issue if we work in AiiDAlab, but I should check it anyways. Thanks!

Reply:
It works if you execute the commands in the IPython shell, but by jolly you have to be patient 😱. I'll just update the commands to the IPython format and remove the Jupyter notebook part.